src/Command/ModuleDeployAfterMagentoCommand.php

```diff
@@ -1,37 +1,106 @@
 <?php
+
 declare(strict_types=1);
 
 namespace App\Command;
 
+use App\CommandQuestion\Question\MysqlContainer;
+use Symfony\Component\Console\Input\InputArgument;
 use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\OutputInterface;
 
 class ModuleDeployAfterMagentoCommand extends AbstractCommand
 {
-    private static $magentoDirectoriesAndFilesToClean = [
-        './app/etc/config.php',
-        './app/etc/config.php',
-        './generated/code/*',
-        './generated/metadata/*',
-        './var/di/*',
-        './var/generation/*',
-        './var/cache/*',
-        './var/page_cache/*',
-        './var/view_preprocessed/*',
-        './pub/static/*',
-
+    private const OPTION_FOLDER = 'folder';
+    private const OPTION_TOGETHER = 'together';
 
+    /**
+     * Cleanup next directories
+     * @var array
+     */
+    private $magentoDirectoriesAndFilesToClean = [
+        'app/etc/config.php',
+        'app/etc/env.php',
+        'app/code/*',
+        'generated/code/*',
+        'generated/metadata/*',
+        'var/di/*',
+        'var/generation/*',
+        'var/cache/*',
+        'var/page_cache/*',
+        'var/view_preprocessed/*',
+        'pub/static/frontend/*',
+        'pub/static/adminhtml/*'
     ];
 
     /**
+     * @var \App\Service\MagentoInstaller $magentoInstaller
+     */
+    private $magentoInstaller;
+
+    /**
+     * ModuleDeployAfterMagentoCommand constructor.
+     * @param \App\Service\MagentoInstaller $magentoInstaller
+     * @param \App\Config\Env $env
+     * @param \App\Service\Shell $shell
+     * @param \App\CommandQuestion\QuestionPool $questionPool
+     * @param null $name
+     */
+    public function __construct(
+        \App\Service\MagentoInstaller $magentoInstaller,
+        \App\Config\Env $env,
+        \App\Service\Shell $shell,
+        \App\CommandQuestion\QuestionPool $questionPool,
+        $name = null
+    ) {
+        parent::__construct($env, $shell, $questionPool, $name);
+        $this->magentoInstaller = $magentoInstaller;
+    }
+
+    /**
+     * @return void
      * @throws \Symfony\Component\Console\Exception\InvalidArgumentException
      */
-    protected function configure()
+    protected function configure(): void
     {
         $this->setName('module:deploy-after-magento')
```

Comment:
@serhiira Such command name makes no sense because there is `together` option. Change the name to something like `magento:test-module-install`. Later `setup:magento` will probably be renamed to `magento:setup` to keep the commands grouped together.